smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -89,21 +89,6 @@ contract MixinKeys is
     _;
   }
 
-  /**
-   * A function which lets the owner of the lock expire a users' key.
-   */
-  function expireKeyFor(
-    address _owner
-  )
-    public
-    onlyOwner
-    hasValidKey(_owner)
-  {
-    Key storage key = keyByOwner[_owner];
-    key.expirationTimestamp = block.timestamp; // Effectively expiring the key
-    emit ExpireKey(key.tokenId);
```

Comment:
I guess the `CancelKey` event in `_cancelAndRefund` replaces this one.

Reply:
It's still in use by `shareKey` today